Gemfile.lock

```diff
@@ -117,8 +117,10 @@ GEM
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
     method_source (1.0.0)
-    mimemagic (0.3.5)
-    mini_mime (1.0.2)
+    mimemagic (0.3.9)
```

Comment:
Needed to update to resolve deprication errors.

Solution found at: [here](https://dev.to/cseeman/what-s-up-with-mimemagic-breaking-everything-he1)